jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenter.java

```diff
@@ -100,7 +101,7 @@ protected DataSetReadyCallback getDataSetReadyCallback(final Integer startRange,
         return errorHandlerBuilder.get().withUUID(tableSettings.getUUID()).withDataSetCallback(
                 dataSet -> {
                     if (dataSet != null && dataSetQueryHelper.getCurrentTableSettings().getKey().equals(tableSettings.getKey())) {
-                        final List<TaskSummary> myTasksFromDataSet = new ArrayList<TaskSummary>();
+                        final LinkedList<TaskSummary> myTasksFromDataSet = new LinkedList<TaskSummary>();
```

Comment:
```suggestion
                        final List<TaskSummary> myTasksFromDataSet = new LinkedList<TaskSummary>();
```

Reply:
This is just to point out that you should always use the interface in the variable declaration (although, as stated in other comments, I vote for replace List by Set)